src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -3265,7 +3240,7 @@ final void poolCloseEventNotify() throws SQLServerException {
             // Also if the app closes a connection handle before committing or rolling back the uncompleted
             // transaction may lock other updates/queries so close the transaction now.
             if (!databaseAutoCommitMode && !(pooledConnectionParent instanceof XAConnection)) {
-                connectionCommand("IF @@TRANCOUNT > 0 ROLLBACK TRAN" /* +close connection */, "close connection");
+                connectionCommand("IF @@TRANCOUNT > 0 ROLLBACK TRAN", "close connection");
```

Comment:
Any reason why we're removing this comment?

Reply:
cos it says "close connection" right after :)